codeworld-api/src/CodeWorld/Driver.hs

```diff
@@ -219,6 +219,9 @@ pictureToDrawing (SolidRectangle _ w h) = Shape $ polygonDrawer (rectangleVertic
 pictureToDrawing (ThickRectangle _ lw w h) = Shape $ pathDrawer (rectangleVertices w h) lw True False
 pictureToDrawing (ClosedCurve _ pts) = Shape $ pathDrawer pts 0 True True
 pictureToDrawing (ThickClosedCurve _ pts w) = Shape $ pathDrawer pts w True True
+pictureToDrawing (Circle _ r) = Shape $ arcDrawer 0 (2 * pi) r 0
+pictureToDrawing (SolidCircle _ r) = Shape $ sectorDrawer 0 (2 * pi) r 
+pictureToDrawing (ThickCircle _ r lw) = Shape $ arcDrawer lw (2 * pi) r 0
```

Comment:
Double check the order of parameters here?  `arcDrawer` has parameters `b`, `e`, `r`, `w`, which should mean "begin angle", "end angle", "radius", and "line width", in that order.